machine-learning-notebooks/transfer-learning-custom-azureml/docker/TFObjectDetectionAPI.dockerfile

```diff
@@ -0,0 +1,86 @@
+#############################################################################################
+# Copyright: Microsoft 2021
```

Comment:
Ugh. Technically we are supposed to put the following header at the top of every source code file in this repository:

```python
# Copyright (c) Microsoft Corporation.
# Licensed under the MIT license.
```

But I'm not so sure that I have been making sure people have been doing that. Anyway, if you wouldn't mind, could you do that for your source files? Thaaaaaanks

Reply:
No problem!  Done.